bson/src/test/unit/org/bson/json/JsonReaderTest.java

```diff
@@ -119,6 +132,16 @@ public void testDateTimeShell() {
     }
 
     @Test
+    public void testDateTimeShellNow() {
+        long currentTime = new Date().getTime();
+        String json = "ISODate()";
```

Comment:
Currently we already support `Date()` and `new Date()` but they are the same.

This just treats `ISODate()` the same - as the logic was missing for the creation of now - hence the ticket by the user.